### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by MMhaidari**
# what is added new
- contact form
- html file  and css file of contact form plus coding and styling 
- form data is linked to formspree page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,276 +1,265 @@
...
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+        <div class="form-elements">
+          <form  action="https://formspree.io/f/mvongqaj"
+          method="POST" action="" class="form">
+            <input type="text" name="user_name" value="Yeremias" />
+            <input type="email" name="user_email" value="natajayanj14@gmail.com" />
+            <textarea name="user_text" id="msg" cols="30" rows="10" placeholder="Write your message here..."></textarea>
```
Comment (MaganaAsiati) [inline index.html:256]: - [ ] Please make sure that the message field (Textarea) accepts up to 500 chars as mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md). You can do that by adding a max length attribute to your fields. To learn more about the attribute, please click [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength). I know you can do that muscle

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @MMhaidari , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [ ] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by MMhaidari** [comment]
# what is added new 
- placeholders added to the inputs
-  character max-length added to input area and textarea

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🥳

Hi @MMhaidari 👋

Well done implementing the changes suggested by previous reviewer. 
**_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Styling is looking great, perfect!
-  ✅ Great work using the right GitHub Flow 
-  ✅ Commits are descriptive, good work.
-  ✅ Linters are working properly, great job! 

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL} I think it would be better if you add all `sections/header` in your readme according to the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md). Right now, your README is missing a lot of headers. I know you can fix that 💪 Keep up the great work

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**